dist/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  <script defer src="runtime.js"></script><script defer src="main.js"></script></head>
+  <body>
+    <section>
+      <h1>Leaderboard</h1>
+      <div class="group">
+        <div class="score-holder">
+          <div class="recent-score">
+            <h3></h3>
+            <button class="refresh">Refresh</button>
+          </div>
+          <div class="score-table"></div>
+        </div>
+        <div class="form-group">
```

Comment:
- [ ] As stated in the project requirement, Kindly consider adding some markup to your page that will be a little bit similar to the provided wireframe. `some hard-coded scores and name` :smile: 
YOur design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201989354-f9556151-6bea-40d9-a1a3-e5e899318346.png)

Template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/201989291-751e5847-21b9-45f4-90ed-8c6b67384f02.png)